assets/pages/bundles-landing/reducers/reducers.js

```diff
@@ -5,6 +5,7 @@
 import { combineReducers } from 'redux';
 
 import { abTestReducer as abTests } from 'helpers/abtest';
+import { intCmpReducer as intCmp } from 'helpers/intCmp';
```

Comment:
Sure, I'm up for an early morning discussion about JavaScript style, sounds like fun 🙉.

I think whether or not you define a preference for default exports depends on the way you tend to construct your js modules. If you commonly have just the one export then I think it makes sense to push the default export style. However, for us most of the time we're exporting multiple things (functions, constants, etc.) from our modules, and the case where we're exporting just the one thing tends to be the exception rather than the rule.

In fact, most of the time this happens because we haven't got around to adding more stuff to the module yet, but chances are it'll have multiple exports in the future. When that happens, you hit the annoyance that I think @svillafe was referring to, where you have to go through and refactor all the places you've imported something, because you've switched from default to non-default exports.

I agree that it would be good to standardise on this, our current linting setup advocates a mix as @svillafe mentioned. However, Dotcom have a [lint rule](https://github.com/guardian/frontend/blob/master/dev/eslint-plugin-guardian-frontend/rules/no-default-export.js) saying that you shouldn't use default exports, and perhaps if we are going to standardise we could follow their lead on this?

🚲🏚